src/Sarif.Viewer.VisualStudio/HashHelper.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography;
+
+namespace Microsoft.Sarif.Viewer
+{
+    internal class HashHelper
```

Comment:
>class [](start = 13, length = 5)

make the class static

Reply:
done

---
In reply to: [606206778](https://github.com/microsoft/sarif-visualstudio-extension/pull/384#discussion_r606206778) [](ancestors = 606206778)